### PR TITLE
Api crud ops

### PR DESCRIPTION
**Pull request by domtunstill**
Post http requests:
Get
Delete
Post

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -7,8 +7,8 @@ import './App.css';
 function App() {
   return (
     <div className="App">
-      <Navbar />
-      <Sidebar />
+      // <Navbar />
```
Comment (melvinlau) [inline src/App.jsx]: Why comment these out?

File: src/components/Post.jsx [hunk 2]
```diff
@@ -20,7 +61,7 @@ class Post extends React.Component{
               Written at: {this.props.timestamp}
             </div>
             <button> Edit </button>
-            <button> Delete </button>
+            <button onClick={this.handleClick}> Delete </button>
```
Comment (melvinlau) [inline src/components/Post.jsx]: might have to rename handleClick to something specific later when Edit button is set up

**Review by melvinlau — APPROVED** [review]

File: src/App.jsx (hunk 1 above)
Comment (John-Dennehy) [reply src/App.jsx]: Agreed

File: src/components/Posts.jsx [hunk 3]
```diff
@@ -48,9 +38,8 @@ class Posts extends React.Component {
           this.state.feed.map((post) => (
             <Post
               id={post.id}
-              author={post.userId}
-              body={post.body}
-              timestamp='Fake time: 14 November 2019 11am'
+              body={post.message}
```
Comment (John-Dennehy) [inline src/components/Posts.jsx:41]: Don't we need userID too?

**Review by melvinlau — APPROVED** [review]

**Review by melvinlau — APPROVED** [review]

**Review by ollyholly — APPROVED** [review]

**Pull request merged**